wlauto/external/uiauto/src/com/arm/wlauto/uiauto/UxPerfUiAutomation.java

```diff
@@ -267,6 +267,16 @@ public void writeResultsToFile(Map<String, Timer> results, String file) throws E
         out.close();
     }
 
```

Comment:
I think this method might be used more than once in when testing on multiple devices - particularly first time runs.